timesketch/api/v1/resources/analysis.py

```diff
@@ -62,6 +63,47 @@ def get(self, sketch_id, timeline_id):
         return self.to_json(analysis_history)
 
 
+class AnalyzerSessionActiveListResource(resources.ResourceMixin, Resource):
```

Comment:
so in the API client you'll see in `sketch.get_analyzer_status` there is a call to `{api_root}/sketches/{sketch.id}/timelines/{timeline_id}/analysis`, which is defined in this file..  that function will go through all timelines in a sketch and do one API call per timeline. My question to you is:

a) can we not just use that already existing API call?

b) If you prefer this one call over one per timeline, can you update the API client to make use of this new function